lfpykernels/kernel_approximator.py

```diff
@@ -414,17 +425,21 @@ def get_kernel(self, probes, Vrest=-65, dt=2**-4,
         # class NetworkCell parameters:
         cellParameters = deepcopy(self.cellParameters)
         cellParameters.update(dict(
-            dt=dt,
-            tstop=t_X + tau,
-            delete_sections=True
-        )
+                dt=dt,
+                tstop=t_X + tau,
+                delete_sections=True
+            )
         )
 
         # Create Cell object representative of whole population
         cell = TemplateCell(**cellParameters)
 
+        self.cell = cell  # saving cell for debugging and plotting
         # set cell rotation
         cell.set_rotation(**self.rotationParameters)
+        # TODO: I COULD FIND NOWHERE THE CELL IS MOVED TO ITS LOCATION??
```

Comment:
Well, this can be explained by that the paper assumed that both populations be centered around 0 along the z-axis.
But for multilayer network I think (but check) that this is still ok, if the distribution that describes the depth distribution accounts for the offset along the z-axis.

Reply:
Assuming that 'loc' in the synapse position arguments is not meant relative to the soma, but rather for example the actual depth of some layer, then I think the cell needs to be moved first, because the cell's z-coordinates are used in the function: get_rand_idx_area_and_distribution_prob. Do you agree?

Reply:
fixed in #71